app/assets/v2/js/pages/fulfill_bounty/token.js

```diff
@@ -68,3 +71,73 @@ fulfillBounty = data => {
     }
   });
 };
+
+
+const is_valid_address = (address) => {
+  switch (web3_type) {
+
+    // etc
+    // celo
+    // rsk
+
+    case 'binance_ext':
+      if (!address.toLowerCase().startsWith('bnb')) {
+        _alert('Enter a valid binance address', 'error');
+        return false;
+      }
+      return true;
+
+    case 'harmony_ext':
+      if (!address.toLowerCase().startsWith('one')) {
+        _alert('Enter a valid harmony address', 'error')
+        return false;
+      }
+      return true;
+
+
+    case 'polkadot_ext':
+      if (address.toLowerCase().startsWith('0x')) {
+        _alert('Enter a valid polkadot address', 'error')
+        return false;
+      }
+      return true;
+
+
+    case 'xinfin_ext':
+      if (!address.toLowerCase().startsWith('xdc')) {
+        _alert('Enter a valid xinfin address', 'error');
+        return false;
+      }
+      return true;
+
+    case 'qr':
+
+      if (token_name == 'BTC') {
+        if (address.toLowerCase().startsWith('0x')) {
```

Comment:
does BTC start with `0x` ?

Reply:
This is displaying the error when a BTC address starts with 0x 👍